lib/src/gpu/applygainmap_gl.cpp

```diff
@@ -235,8 +235,9 @@ bool isBufferDataContiguous(uhdr_raw_image_t* img) {
     uint16_t* y = static_cast<uint16_t*>(img->planes[UHDR_PLANE_Y]);
     uint16_t* u = static_cast<uint16_t*>(img->planes[UHDR_PLANE_UV]);
     std::ptrdiff_t sz = u - y;
+    long pixels = img->w * img->h;
```

Comment:
Do we need to consider stride here?

Reply:
Here we are checking if the buffer data is contiguous. That is it has no horizontal stride (width and stride are same) and there is no gap between y/u/v buffers. To determine there is no gap between y,u,v buffers we are comparing u ptr - y ptr with total number of pixels. So width is appropriate here.